WordPress/gutenberg
# Scripts: use default babel if none is found in project

Changes to make Webpack use the `@wordpress/babel-preset-default` babel config if none is found when running `wp-scripts build/start`.

## Test

- make sure you can build Gutenberg for development: `npm run dev` and that it works as expected.
- make sure you can build Gutenberg for production: `npm run build` and that it works as expected.
- external testbed using the default babel config https://github.com/nosolosw/understanding-gutenberg/pull/7
